### PR TITLE
Added tags and descriptions for platform based packages

### PR DESCRIPTION
**Pull request by pMaske**
Added tags and descriptions to newly created nuget packages (per platform)

### CODE REVIEW

File: src/ARToolkit.Maui/Esri.ArcGISRuntime.ARToolkit.Maui.csproj [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net6.0;$(NetAndroidTargetFramework);$(NetiOSTargetFramework)</TargetFrameworks>
+	<Description>ArcGIS Runtime Augmented Reality controls and utilities for .NET MAUI apps including .NET for Android, .NET for iOS, .NET for macOS, or Windows UI 3 (WinUI 3)</Description>
```
Comment (dotMorten) [inline src/ARToolkit.Maui/Esri.ArcGISRuntime.ARToolkit.Maui.csproj:5]: only android and ios. No macos or winui support.

File: src/ARToolkit.Maui/Esri.ArcGISRuntime.ARToolkit.Maui.csproj [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+    <PackageTags>ArcGIS Cartography Geo Geographic Geography Geolocation Geospatial GIS Latitude Location Longitude Map Mapping Maps Places Spatial Augmented Reality AR 3D .NET MAUI Android iOS WinUI Mac Catalyst MacCatalyst toolkit</PackageTags>
```
Comment (dotMorten) [inline src/ARToolkit.Maui/Esri.ArcGISRuntime.ARToolkit.Maui.csproj:6]: same: Remove unsupported platforms

File: src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFrameworks>net6.0;$(NetAndroidTargetFramework);$(NetiOSTargetFramework)</TargetFrameworks>
+		<Description>ArcGIS Runtime controls and utilities for .NET MAUI apps including .NET for Android, .NET for iOS, .NET for macOS, or Windows UI 3 (WinUI 3)</Description>
```
Comment (dotMorten) [inline src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj:5]: Should probably be .NET for MacCatalyst?

Comment (pMaske) [reply src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj:5]: @dotMorten I used based how it was [referred ](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui#how-net-maui-works)in this microsoft doc. I can change if you think `.NET for MacCatalyst` is more correct

**Review by dotMorten — APPROVED** [review]

**Pull request merged**